# HG changeset patch
# User Rik <rik@octave.org>
# Date 1354814195 28800
#      Thu Dec 06 09:16:35 2012 -0800
# Node ID 5c40e5b9cef24dd4f1a95dfa79a58188f85a40e5
# Parent  628011b1bd2f5a9c7ed7fa1f0e093e4397972b59
maint: Remove unnecessary execute permissions from java images in doc directory.

* doc/interpreter/java-images/image001.png,
doc/interpreter/java-images/image002.png,
doc/interpreter/java-images/image003.png,
doc/interpreter/java-images/image004.png,
doc/interpreter/java-images/image005.png,
doc/interpreter/java-images/image006.png,
doc/interpreter/java-images/image007.png,
doc/interpreter/java-images/image008.png,
doc/interpreter/java-images/image009.png:
Remove unnecessary execute permissions image files.

diff --git a/doc/interpreter/java-images/image001.png b/doc/interpreter/java-images/image001.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image002.png b/doc/interpreter/java-images/image002.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image003.png b/doc/interpreter/java-images/image003.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image004.png b/doc/interpreter/java-images/image004.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image005.png b/doc/interpreter/java-images/image005.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image006.png b/doc/interpreter/java-images/image006.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image007.png b/doc/interpreter/java-images/image007.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image008.png b/doc/interpreter/java-images/image008.png
old mode 100755
new mode 100644
diff --git a/doc/interpreter/java-images/image009.png b/doc/interpreter/java-images/image009.png
old mode 100755
new mode 100644
